pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -1299,37 +1299,31 @@ func NewTimestampedCheckpointFileName() string {
 }
 
 // FetchPreviousBackups takes a list of URIs of previous backups and returns
-// their manifest as well as the encryption options of the first backup in the
-// chain.
+// their manifests. The caller is expected to pass in the fully hydrated
+// encryptionParams required to read the manifests.
 func FetchPreviousBackups(
```

Comment:
I question the need for this method anymore but wanted to keep the diff small, let me know if you'd rather i rip it out and export `GetBackupManifests`.

Reply:
Given that they only differ by a tracing span now, I think we should rip one of them out.  We are also updating the only call site of this function, so hopefully the diff isn't affected too much.